opt/migrate-database.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -exuo pipefail
+
+# This script is intended to be run in a container. Its job is to migrate the PGSQL database to a newer version.
+
+# The following variables are required:
+DB_FOLDER=/home/${SYSTEM_USER}/.postgresql
+
+OLD_PGSQL_DB_VERSION=`cat ${DB_FOLDER}/PG_VERSION`  # The version of the database before the upgrade is taken from the PG_VERSION file.
+NEW_PGSQL_DB_VERSION=`conda run -n pgsql psql -V | awk '{ print int( $3 ) }'`  # The new version of PGSQL is given by the `psql -V` command installed in the pgsql conda environment.
+
+TIMESTAMP=$(date +%Y%m%d%H%M%S)
+BACKUP_DB_FOLDER=/home/${SYSTEM_USER}/.postgresql-bak-${TIMESTAMP}
+OLD_DB_CONDA_ENV_NAME="pgsql-backup-${OLD_PGSQL_DB_VERSION}-${TIMESTAMP}"
+DUMP_FILE_NAME="/home/${SYSTEM_USER}/pgsql-dump-${OLD_PGSQL_DB_VERSION}-${TIMESTAMP}.sql"
+
+# Make sure the new PostgreSQL server is shut down.
+conda run -n pgsql pg_ctl -D ${DB_FOLDER} stop || true # Ignore errors, as the database might not be running.
+
+# Part 1: dump the old database.
+
+# Make a backup of the database.
+mv ${DB_FOLDER} ${BACKUP_DB_FOLDER}
```

Comment:
Where would this script show it's output?
I would put an echo here to show where the backup is

(although when the script is executed in the debug mode (`set -x`), each command is echoed, so I guess and extra echo make it a bit more clear what is happening)
```suggestion
echo "Creating a DB backup to ${BACKUP_DB_FOLDER}"
mv ${DB_FOLDER} ${BACKUP_DB_FOLDER}
```

Reply:
If we can retrieve the debug information, yes, I agree it is more clear to print this massage. But I don't know where to check these printed-out messages. Is it write to some log files?